### PR TITLE
DOCS-1140 NDM Troubleshooting

### PR DESCRIPTION
**Pull request by ruthnaebeck**
### What does this PR do?
Add Network Device Monitoring Troubleshooting page

### Motivation
Jira request

### Preview link
https://docs-staging.datadoghq.com/ruth/docs-1140/network_performance_monitoring/devices/troubleshooting

### Additional Notes
PM approval needed to merge

### CODE REVIEW

File: content/en/network_performance_monitoring/devices/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+**sysOID** - System object identifier - A specific address that defines the device type. All devices have a unique ID that defines it. For example, the Meraki base sysOID is `1.3.6.1.4.1.29671`.
+
+**MIB** - Managed information base - A database or list of all the possible OIDs and their definitions that are related to the MIB. For example, the `IF-MIB` (interface MIB) contains all the OIDs for descriptive information about a device's interface.
+
+## FAQ
+
+#### What SNMP versions does Datadog support?
+
+Datadog supports all three versions of SNMP: SNMPv1, SNMPv2 and SNMPv3.
```
Comment (cswatt) [inline content/en/network_performance_monitoring/devices/troubleshooting.md:28]:
```suggestion
Datadog supports all three versions of SNMP: SNMPv1, SNMPv2, and SNMPv3.
```

File: content/en/network_performance_monitoring/devices/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+If you send a feature request, Datadog support needs a `snmpwalk` from the requested device. Run the following and send the output:
+
+```
+snmpwalk -v2 -c -Obef <COMMUNITY_STRING> <IP_ADDRESS> 1.3.6
+```
+
+#### Why am I only seeing one metric collected for my networks and it’s the number of devices collected at zero?
+
+1. Try loosening ACLs/firewall rules for your devices.
+2. Run `snmpwalk -v2c -ObentU -c <COMMUNITY_STRING> <IP_ADDRESS> 1.3.6` from the host your Agent is running on. If you get a timeout without any response, there is likely something blocking the Datadog Agent from collecting metrics from your device.
```
Comment (AlexandreYang) [inline content/en/network_performance_monitoring/devices/troubleshooting.md:67]:
```suggestion
2. Run `snmpwalk -O bentU -v 2c -c <COMMUNITY_STRING> <IP_ADDRESS>:<PORT> 1.3.6` from the host your Agent is running on. If you get a timeout without any response, there is likely something blocking the Datadog Agent from collecting metrics from your device.
```

Same as above.

**Review by AlexandreYang — CHANGES_REQUESTED** [review]
Looks great! Thx!

Suggested a fix for the `snmpwalk` command.

File: content/en/network_performance_monitoring/devices/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+```
+
+#### Can I still use Network Device Monitoring if my device-model pair isn't supported?
+
+Datadog collects generic base-line metrics from all devices. If there are unsupported metrics from a vendor MIB, you can write a custom profile, or send a feature request to [Datadog support][1].
+
+If you send a feature request, Datadog support needs a `snmpwalk` from the requested device. Run the following and send the output:
+
+```
+snmpwalk -v2 -c -Obef <COMMUNITY_STRING> <IP_ADDRESS> 1.3.6
```
[the remark this answers is not in the document]
Comment (AlexandreYang) [reply content/en/network_performance_monitoring/devices/troubleshooting.md:61]: Details about the `snmpwalk` options:

```
USAGE: snmpwalk [OPTIONS] AGENT [OID]

  Version:  5.6.2.1
  Web:      http://www.net-snmp.org/
  Email:    net-snmp-coders@lists.sourceforge.net

OPTIONS:
  -h, --help		display this help message
  -H			display configuration file directives understood
  -v 1|2c|3		specifies SNMP version to use
  -V, --version		display package version number
SNMP Version 1 or 2c specific
  -c COMMUNITY		set the community string
SNMP Version 3 specific
  -a PROTOCOL		set authentication protocol (MD5|SHA)
  -A PASSPHRASE		set authentication protocol pass phrase
  -e ENGINE-ID		set security engine ID (e.g. 800000020109840301)
  -E ENGINE-ID		set context engine ID (e.g. 800000020109840301)
  -l LEVEL		set security level (noAuthNoPriv|authNoPriv|authPriv)
  -n CONTEXT		set context name (e.g. bridge1)
  -u USER-NAME		set security name (e.g. bert)
  -x PROTOCOL		set privacy protocol (DES|AES)
  -X PASSPHRASE		set privacy protocol pass phrase
  -Z BOOTS,TIME		set destination engine boots/time
General communication options
  -r RETRIES		set the number of retries
  -t TIMEOUT		set the request timeout (in seconds)
Debugging
  -d			dump input/output packets in hexadecimal
  -D[TOKEN[,...]]	turn on debugging output for the specified TOKENs
			   (ALL gives extremely verbose debugging output)
General options
  -m MIB[:...]		load given list of MIBs (ALL loads everything)
  -M DIR[:...]		look in given list of directories for MIBs
    (default: $HOME/.snmp/mibs:/usr/local/share/snmp/mibs:/usr/share/snmp/mibs)
  -P MIBOPTS		Toggle various defaults controlling MIB parsing:
			  u:  allow the use of underlines in MIB symbols
			  c:  disallow the use of "--" to terminate comments
			  d:  save the DESCRIPTIONs of the MIB objects
			  e:  disable errors when MIB symbols conflict
			  w:  enable warnings when MIB symbols conflict
			  W:  enable detailed warnings when MIB symbols conflict
			  R:  replace MIB symbols from latest module
  -O OUTOPTS		Toggle various defaults controlling output display:
			  0:  print leading 0 for single-digit hex characters
			  a:  print all strings in ascii format
			  b:  do not break OID indexes down
			  e:  print enums numerically
			  E:  escape quotes in string indices
			  f:  print full OIDs on output
			  n:  print OIDs numerically
			  q:  quick print for easier parsing
			  Q:  quick print with equal-signs
			  s:  print only last symbolic element of OID
			  S:  print MIB module-id plus last element
			  t:  print timeticks unparsed as numeric integers
			  T:  print human-readable text along with hex strings
			  u:  print OIDs using UCD-style prefix suppression
			  U:  don't print units
			  v:  print values only (not OID = value)
			  x:  print all strings in hex format
			  X:  extended index format
  -I INOPTS		Toggle various defaults controlling input parsing:
			  b:  do best/regex matching to find a MIB node
			  h:  don't apply DISPLAY-HINTs
			  r:  do not check values for range/type legality
			  R:  do random access to OID labels
			  u:  top-level OIDs must have '.' prefix (UCD-style)
			  s SUFFIX:  Append all textual OIDs with SUFFIX before parsing
			  S PREFIX:  Prepend all textual OIDs with PREFIX before parsing
  -L LOGOPTS		Toggle various defaults controlling logging:
			  e:           log to standard error
			  o:           log to standard output
			  n:           don't log at all
			  f file:      log to the specified file
			  s facility:  log to syslog (via the specified facility)

			  (variants)
			  [EON] pri:   log to standard error, output or /dev/null for level 'pri' and above
			  [EON] p1-p2: log to standard error, output or /dev/null for levels 'p1' to 'p2'
			  [FS] pri token:    log to file/syslog for level 'pri' and above
			  [FS] p1-p2 token:  log to file/syslog for levels 'p1' to 'p2'
  -C APPOPTS		Set various application specific behaviours:
			  p:  print the number of variables found
			  i:  include given OID in the search range
			  I:  don't include the given OID, even if no results are returned
			  c:  do not check returned OIDs are increasing
			  t:  Display wall-clock time to complete the request
			  E {OID}:  End the walk at the specified OID
```

**Pull request merged**